indico/modules/events/abstracts/fields.py

```diff
@@ -30,13 +29,28 @@
 from indico.modules.events.abstracts.util import serialize_abstract_person_link
 from indico.modules.events.contributions.models.persons import AuthorType
 from indico.modules.events.fields import PersonLinkListFieldBase
+from indico.modules.users.models.users import User
 from indico.util.decorators import classproperty
 from indico.util.i18n import _
 from indico.web.forms.base import AjaxFieldMixin
 from indico.web.forms.fields import MultipleItemsField, JSONField
 from indico.web.forms.widgets import JinjaWidget, SelectizeWidget
 
 
+def serialize_user(user):
```

Comment:
I'm pretty much against static methods in Python (with very few exceptions). I'd underscore it.